text/src/autogluon/text/automm/predictor.py

```diff
@@ -452,7 +458,8 @@ def fit(
         self._df_preprocessor = df_preprocessor
         self._data_processors = data_processors
         self._model = model
-        self.mixup_fn = mixup_fn
+        self._mixup_fn = mixup_fn
+        self._loss_func = loss_func
```

Comment:
There is no need to have these two attributes since they are not saved.

Reply:
Yeah, I just see `self._loss_func` is used below, but `self._mixup_fn` seem not necessary.